webpack/index.d.ts

```diff
@@ -10,8 +10,8 @@ declare namespace webpack {
     interface Configuration {
         context?: string;
         entry?: string | string[] | Entry;
-        /** Choose a developer tool to enhance debugging. */
-        devtool?: string;
+        /** Choose a style of source mapping to enhance the debugging process. These values can affect build and rebuild speed dramatically. */
+        devtool?: 'eval' | 'inline-source-map' | 'cheap-eval-source-map' | 'cheap-source-map' | 'cheap-module-eval-source-map' | 'cheap-module-source-map' | 'eval-source-map' | 'source-map' | 'nosources-source-map' | 'hidden-source-map' | 'nosources-source-map' | boolean;
```

Comment:
Added them to the union. This gives better safety than `string`, at the cost of typings legibility. IMO the tradeoff is worth it - what do you think?